!SESSION 2015-09-20 17:08:45.025 -----------------------------------------------
eclipse.buildId=4.5.0.I20150603-2000
java.version=1.8.0_60
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.jee.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.jee.product

!ENTRY org.eclipse.egit.ui 2 0 2015-09-20 17:09:36.710
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\La Familia'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.egit.core 4 0 2015-09-20 18:33:41.052
!MESSAGE An exception occurred during push on URI https://github.com/CSantosCompSci/RunWaySim.git: An internal Exception occurred during push: Source ref refs/heads/master doesn't resolve to any object.
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:182)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:228)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:167)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:229)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Source ref refs/heads/master doesn't resolve to any object.
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:288)
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:192)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:646)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:1269)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:153)
	... 4 more

!ENTRY org.eclipse.egit.core 4 0 2015-09-20 18:34:10.278
!MESSAGE An exception occurred during push on URI https://github.com/CSantosCompSci/RunWaySim.git: An internal Exception occurred during push: Source ref refs/heads/master doesn't resolve to any object.
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:182)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:228)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:167)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:229)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Source ref refs/heads/master doesn't resolve to any object.
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:288)
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:192)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:646)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:1269)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:153)
	... 4 more

!ENTRY org.eclipse.egit.ui 4 0 2015-09-20 18:35:31.879
!MESSAGE Exception caught during execution of add command
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:211)
	at org.eclipse.egit.core.op.CommitOperation.addUntracked(CommitOperation.java:222)
	at org.eclipse.egit.core.op.CommitOperation.access$4(CommitOperation.java:210)
	at org.eclipse.egit.core.op.CommitOperation$1.run(CommitOperation.java:195)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2241)
	at org.eclipse.egit.core.op.CommitOperation.execute(CommitOperation.java:206)
	at org.eclipse.egit.ui.internal.commit.CommitJob.run(CommitJob.java:98)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(Unknown Source)
	at org.eclipse.jgit.util.IO.readFully(IO.java:302)
	at org.eclipse.jgit.internal.storage.file.ObjectDirectoryInserter.insert(ObjectDirectoryInserter.java:103)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:187)
	... 7 more
